Appraisals

```diff
@@ -28,6 +28,11 @@ appraise "rails-7.0" do
   gem "tailwindcss-rails", "~> 2.0"
 end
 
+appraise "rails-7.1" do
+  gem "rails", "~> 7.1.1"
```

Comment:
I think we can use the base version here (similarly to what we do for older versions above in the same file)

```suggestion
  gem "rails", "~> 7.1.0"
```